pkgs/tools/misc/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, fetchFromGitHub, rustPlatform, cmake, perl, pkgconfig, zlib
+, darwin, libiconv
+}:
+
+with rustPlatform;
+
+buildRustPackage rec {
+  name = "sd-${version}";
+  version = "0.4.3";
+
+  cargoSha256 = "08zzn3a32xfjkmpawcjppn1mr26ws3iv40cckiz8ldz4qc8y9gdh";
+
+  src = fetchFromGitHub {
+    owner = "chmln";
+    repo = "sd";
+    rev = "v${version}";
+    sha256 = "0jy11a3xfnfnmyw1kjmv4ffavhijs8c940kw24vafklnacx5n88m";
```

Comment:
This hash is also from the `exa` package. You should be able to get the correct hash with this command:

```bash
nix run nixpkgs.nix-prefetch-git -c nix-prefetch-git https://github.com/chmln/sd 0.4.3
```